src/components/FeaturedContent/FeaturedContent.tsx

```diff
@@ -0,0 +1,112 @@
+import { Box, chakra, useMultiStyleConfig } from "@chakra-ui/react";
+import React, { forwardRef } from "react";
+import { ImageProps } from "../Image/Image";
+
+export const featuredContentWidthArray = [
+  "oneQuarter",
+  "oneThird",
+  "oneHalf",
+  "twoThirds",
+  "threeQuarters",
+];
+export type featuredContentWidthType = typeof featuredContentWidthArray[number];
+
+export const featuredContentPositionArray = ["start", "end"] as const;
+export type featuredContentPositionType =
+  typeof featuredContentPositionArray[number];
+
+interface FeaturedContentImageProps extends ImageProps {
+  /** String value that specifies the width of the image rendered within the component. */
+  width?: featuredContentWidthType;
+  /** String value that specifies the position of the image rendered within the component. */
+  position?: featuredContentPositionType;
+}
+
+export interface FeaturedContentProps {
+  /** Optional CSS class name to add. */
+  className?: string;
+  /** ID that other components can cross reference for accessibility purposes. */
+  id?: string;
+  /** The text content rendered in the component.  DS components and native HTML can be passed in this prop. */
+  textContent: string | JSX.Element;
+  /** Whether component will fill the full width of the browser window, instead of just its parent element.
+   * False by default. */
+  isFullWidth: boolean;
+  /** Data object that contains the props related to the image element: alt, position, src, width.  */
+  imageProps: FeaturedContentImageProps;
+}
+
+/**
+ * The FeaturedContentImage component is used internally in the `FeaturedContent` component. It
+ * renders an `Image` component but with overriding styles specific to the
+ * `FeaturedContent` component.
+ */
+function FeaturedContentImage(
+  props: React.ComponentProps<"img"> & FeaturedContentImageProps
+) {
+  const { alt, src } = props;
+  return <img alt={alt} src={src} style={{ display: "none" }} />;
```

Comment:
Just curious, did you try using the DS `Image` component and found that didn't scale as well as this `img` element?

Reply:
Yes, so I originally tried with the`Image` component but like you noted, there's props on that that interfered with the variable width display in FeaturedContent. Using the `img` element allowed me to style it much more straightforwardly (though I can definitely see the merit of reusing components wherever possible).

Reply:
That makes sense. We've encountered similar friction in other components when we had custom values for props that expect a certain set of values (especially with color and background color).

In this case, `imageProps.width` is used for the wrapper and the theme object but not in the image element itself. I think the intersection type for `props` is causing the conflict for the `width` HTML attribute and react prop. Try removing `React.ComponentProps<"img">`, I'm not sure it's needed if this only needs src and alt. Let's also go through this together at some point.

Reply:
This does work the same without `React.ComponentProps<"img">`, so just took it out. It would definitely be helpful to step through each layer of this with someone familiar with the `Image` component.